api/live.js

```diff
@@ -96,153 +92,134 @@ async function handleContrib(userId, req, res) {
 
   if (!input.taskId) {
     return res.status(400).json({ error: "Invalid request" });
-  } else {
-    try {
-      const taskMultiplier = getMultiplierForTask(input.taskId);
-      const [result, isCreated] = await models.Contribution.findOrCreate({
-        defaults: {
-          multiplier: taskMultiplier,
-          scannerId: req.user.id
-        },
-        where: {
-          personId: userId,
-          taskId: input.taskId
-        }
-      });
-      if (!isCreated) {
-        return res.status(400).json({
-          error: "User already has completed this task"
-        });
-      }
-      return res.json({
-        contribution: result,
-        message: `Successfully created a task contribution for ${profile.firstName} ${profile.lastName}`
-      });
-    } catch (e) {
-      return res.status(500).json({ error: e.message });
+  }
+  const taskMultiplier = getMultiplierForTask(input.taskId);
+  const [result, isCreated] = await models.Contribution.findOrCreate({
+    defaults: {
+      multiplier: taskMultiplier,
+      scannerId: req.user.id
+    },
+    where: {
+      personId: userId,
+      taskId: input.taskId
     }
+  });
+  if (!isCreated) {
+    return res.status(400).json({
+      error: "User already has completed this task"
+    });
   }
+  return res.json({
+    success: {
+      contribution: result,
+      message: `Successfully created a task contribution for ${profile.firstName} ${profile.lastName}`
+    }
+  });
 }
 
 async function getMultiplierForTask(taskId) {
-  try {
-    const result = await models.Task.findOne({
-      where: {
-        id: taskId
-      },
+  const result = await models.Task.findOne({
+    where: {
+      id: taskId
+    },
 
-      include: [
-        {
-          model: models.Grouping,
-          required: false,
-          include: [
-            {
-              model: models.Multiplier,
-              required: false
-            }
-          ]
-        }
-      ]
-    });
-    const multipliers = result.get("Grouping").get("Multipliers");
-    return multipliers.reduce((x, y) => x + y.dataValues.multiplierValue, 0);
-  } catch (e) {
-    Sentry.captureMessage(e.message);
```

Comment:
Wait don't delete the try catch here, this uses failures to assume the default multiplier value, and doesn't actually represent a potential user failure.